event_routing_backends/tests/test_models.py

```diff
@@ -39,50 +67,43 @@ def test_enabled_router_is_returned(self):
         TieredCache.dangerous_clear_all_tiers()
         self.assertEqual(RouterConfiguration.get_enabled_routers('second')[0], second_router)
 
-    def test_allowed_hosts(self):
+    @ddt.data(
+        ({'context.org_id': 'test'}, True),
+        ({'non_existing.id.value': 'test'}, False),
+        ({'context.org_id': 'abc', 'name': None}, False),
+        ({'context.org_id': 'test', 'name': ['problem_check', 'showanswer', 'stop_video']}, True),
```

Comment:
@rehanedly could you please one more test
`({'context.org_id': 'abc', 'name': 'problem_check'}, True),`

Reply:
@ziafazal it would be ({'context.org_id': 'abc', 'name': 'problem_check'}, False), because we are passing org_id test in actual event

Reply:
@ziafazal I have added some regax expression test cases